guidelines/releases/release-strategy.md

```diff
@@ -0,0 +1,173 @@
+
+### Development end
+This is the last day of the period planned for the release development.
+
+### Release decision
+The Release Manager and the Product Leadership decide to publish the release, based on the status and readiness of the developed functionality. The Release Manager communicates this decision before the release execution starts.
+
+### Release execution
+The Release Engineer executes the release at any time of producing the release artifact. This could result in a release candidate or a final release that is not an automated nightly or weekly build. The release execution begins when the process starts and finishes after publishing all artifacts and the related documentation.
+
+During the execution of a release, Kyma developers run all end-to-end test scenarios. If any single scenario fails, the Release Engineer cannot publish the release.
+
+The Release Manager or Kyma developers notify the Kyma community when the release candidate is available. They pass this information to the public on Slack, requesting the community to validate and provide their short-term feedback. The deadline for the feedback is the day before the planned final release. Ideally, the final release is the same as the last release candidate, avoiding any additional changes. If the Kyma developers identify any release-related issues, they asses them and decide together with the Release Manager if they must fix them and produce a new release candidate.
+
+### Release publishing
+The final release is available in the GitHub releases, including the installation instructions. It also includes the complete changelog that lists all pull requests merged into this release.
+
+A Technical Writer publishes the blog post on the public Kyma website to announce the release. The post includes release notes prepared based on the input from the Product Owners.
+
+You can learn about the new release from additional notifications published on social media, Slack channels, or announcements made during the Core SIG meetings. Alternatively, you can [subscribe](./release-subscription.md) to GitHub notifications and receive updates each time a new release is available.
+
+## Release scope
+For each of the planning periods described in the release schedule, GitHub epics and issues define and document the release scope with regards to functionality, corrections, and even non-functional requirements. The collection of all documented issues within a release represents the expected scope of that release that the whole organization and all teams define and commit to. The corresponding release in ZenHub identifies all issues and epics that fall under the final release scope.
+
+When planning the release scope, all persons involved in the release must take into consideration the GDPR requirements when processing or storing personal data.
+
+### Down-porting
```

Comment:
I think we should use the term "downgrade". "Downporting" doesn't really have a definition and is used scarcely. 

![image](https://user-images.githubusercontent.com/29116492/52855820-d5f89d00-3122-11e9-8aaa-4c6e1636c9f8.png)

vs

![image](https://user-images.githubusercontent.com/29116492/52855906-1a843880-3123-11e9-9b6b-69bbb6e60bcd.png)

Reply:
I rephrased that a bit.